doc/source/reference/arrays.datetime.rst

```diff
@@ -36,6 +36,11 @@ letters, for a "Not A Time" value.
 
     >>> np.datetime64('2005-02-25')
     numpy.datetime64('2005-02-25')
+    
+    UNIX time, by providing integer and a date unit :
```

Comment:
UNIX isn't related to "time", only to "epoch". So how about:

_From an integer and a date unit, here 1 year since the UNIX epoch:_